translator/GCP_cloud_function/cloud_function/icone_translator.py

```diff
@@ -1,16 +1,12 @@
 import xmltodict
```

Comment:
It doesn't look like we're using xmltodict in this file, can we remove?

Reply:
we removed it.